hpx/hpx_main.hpp

```diff
@@ -9,6 +10,18 @@
 #include <hpx/config.hpp>
 #include <hpx/hpx_init.hpp>
 
+// We support different implementation depending upon the Operating
+// System in use.
+#if defined(__linux__)
+
+// include_libhpx_wrap here is an override for the one present in
+// src/hpx_wrap.cpp. The value of this variable defines if we need
+// to change the program's entry point or not.
+extern int include_libhpx_wrap;
+int include_libhpx_wrap = 1;
+
```

Comment:
Could you place these global variables into a separate namespace, please? In other spots we use `namespace hpx_startup {}` for this.